xbmc/filesystem/CurlFile.cpp

```diff
@@ -593,6 +593,10 @@ void CCurlFile::SetCommonOptions(CReadState* state)
   if( m_contentencoding.length() > 0 )
     g_curlInterface.easy_setopt(h, CURLOPT_ENCODING, m_contentencoding.c_str());
 
+  // setup Accept-Encoding if requested
+  if (m_acceptencoding.length() > 0)
+    g_curlInterface.easy_setopt(h, CURLOPT_ACCEPT_ENCODING, m_contentencoding.c_str());
```

Comment:
I can try to find time (but I don't think I have a current build env (although for this small a change I probably just need to do the git dance to get a change...))